### ISSUE TITLE
Select: Disabled with selected value icons issue

### ISSUE THREAD

**Issue by raplemie** [iTwin/iTwinUI#395]
#### Describe the bug (actual behavior)
Both "open" and "close" icons are visible in the Disabled With Selected Value Select example

![image](https://user-images.githubusercontent.com/1904889/144073999-d7b727b3-6212-4cd4-99d3-c8f0bd176183.png)

#### Expected behavior
Only 1 icon should be visible

#### Reproduction
Link to a minimal repro: https://itwin.github.io/iTwinUI-react/?path=/story/input-select--disabled-with-selected-value

##### Steps to reproduce
1. Open the provided link
1. See the result
(I did not write this example, nor have I read the code for it, so I dont know what exactly is causing the issue)

#### Additional information
Keep up the good work !

**Comment by mayank99** [comment]
Good catch. Moving this issue to CSS repo. We might actually take this opportunity to make the icon consistent with ComboBox too
